docs/.vuepress/theme/components/Home.vue

```diff
@@ -785,11 +785,11 @@
       </div>
     </div>
   </section>
-  <section class="d-bgc-purple-500 d-py64 d-ta-center">
-    <div class="d-headline-large d-fw-medium d-fc-primary-inverted d-px48 d-flow16 d-d-flex d-jc-center d-ai-center">
+  <section class="d-bt d-bbw1 d-bc-subtle d-bgc-secondary d-py64 d-ta-center">
```

Comment:
Remove class `d-bbw1`. 

It is not necessary here as it doesn't even have border at the bottom, and border-top-width default value is 1.

```suggestion
  <section class="d-bt d-bc-subtle d-bgc-secondary d-py64 d-ta-center">
```